### ISSUE TITLE
research validation during API key signup

### ISSUE THREAD

**Issue by jason-upchurch** [fecgov/openFEC#4306]
### Summary
We need to research how to appropriately validate users requesting API keys. This work may in part depend on #4305 

### Completion criteria
- [ ] have meeting to discuss what "validation" looks like for our API users
- [ ] implement or open appropriate tickets to implement solution

**Comment by JonellaCulmer** [comment]
Closing in favor of ticket in fec-accounts.
